site.yml

```diff
@@ -157,6 +157,14 @@
       tags: ['ceph', 'ceph-osd']
   environment: "{{ env_vars|default({}) }}"
 
+- name: ceph update
+  hosts: ceph_monitors:ceph_osds:controller:compute
```

Comment:
So previously, within the `tasks/main.yml` of the `ceph-common` role, whenever the `ceph.conf` file was changed, the template play would update this file and would notify the restart handlers to actually enforce these changes on the mons+osds+ctrls+compute nodes. This role essentially does the same thing but now that I think about it, I can make this role run as a dependency (in the roles invoked by the `ceph` tag) right after `ceph-common` so that it doesn't have to be called from `site.yml` :/... I'll test this out tomorrow because that would be a better invocation of this role.